internal/server/auth/authorization.go

```diff
@@ -3,6 +3,7 @@ package auth
 import (
 	"context"
 	"fmt"
+	"github.com/lxc/incus/v6/shared/api"
```

Comment:
Incorrect ordering of imports, the project-local imports go in the last chunk of imports.